src/meson.build

```diff
@@ -114,7 +114,7 @@ git_branch_info = vcs_tag(
 # First build a static library of all sources, so that it can be reused
 # in tests
 libwayfire_sta = static_library('libwayfire', wayfire_sources,
-    dependencies: wayfire_dependencies,
+    dependencies: [wayfire_dependencies, json.as_link_whole()],
```

Comment:
why couldn't we use .as_link_whole() in the list above?

Reply:
The answer is in the Note above.